src/pages/student_list/StudentList.styles.ts

```diff
@@ -1,10 +1,20 @@
 import styled from "styled-components";
+import {
+  BACKGROUND_COLOR,
+  BORDER_RADIUS,
+  ERROR_COLOR,
+  SUCCESS_COLOR,
+  TERTIARY_COLOR,
+} from "../../utils/Stylesheet";
 export const StudentsContainer = styled.div`
   width: 100%;
   padding-top: 20px;
   min-height: 100vh;
 `;
-
+export const Title = styled.h1`
+  font-size: 24px;
+  margin-left: 80px;
```

Comment:
Falta el font-family. Acordate de agregar en el stylesheet lo del weight porque no está acá
```suggestion
  font-size: 24px;
  margin-left: 80px;
  font-family: ${FONT_FAMILY};
  font-weight: ${FONT_WEIGHT_BOLD};
```